GREEDY.UnitTests/WritingToFileControllerUnitTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using GREEDY.Models;
+using GREEDY.Controllers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GREEDY.UnitTests
+{
+    [TestClass]
+    public class WritingToFileControllerUnitTest
+    {
+        [DataTestMethod]
+        [DataRow("")]
+        [DataRow("   ")]
+        [DataRow("123")]
+        public void WriteToFileTest (string data)
+        {
+            //arrange
+            Receipt receipt = new Receipt
+            {
+                LinesOfText = new List<String>() { data }
+            };
+            string filePath = "testReceipt.txt";
+            //act
+            new WritingToFileController().WriteToFile(filePath, receipt);
```

Comment:
If I correct understand, we have to have another unit test to get information from receipt class and to write to file. For this unit test we use two classes. If I am correct, it is integration testing.